src/Tag/Tag.tsx

```diff
@@ -20,20 +21,30 @@ export const Tag: FC<TagProps> = ({
   bgColor,
   className,
   typo,
+  bgGradient = false,
 }) => (
-  <Wrapper bgColor={bgColor} className={className} borderColor={borderColor}>
+  <Wrapper
+    bgColor={bgColor}
+    className={className}
+    borderColor={borderColor}
+    bgGradient={bgGradient}
+  >
     <TagText tag="span" typo={typo ?? 'label'} color={color}>
       {label}
     </TagText>
   </Wrapper>
 )
 
-type WrapperProps = Pick<TagProps, 'bgColor' | 'borderColor'>
+type WrapperProps = Pick<TagProps, 'bgColor' | 'borderColor' | 'bgGradient'>
 
 const Wrapper = styled.div<WrapperProps>`
-  background-color: ${({ bgColor }) => theme.colors[bgColor]};
+  background-color: ${({ bgColor }) => bgColor && theme.colors[bgColor]};
+  background: ${({ bgGradient }) =>
+    bgGradient &&
+    `linear-gradient(90deg, rgba(247, 46, 73, 1) 0%, rgba(246, 148, 210, 1) 100%)`};
   border-radius: 8px;
-  border: 1px solid ${({ borderColor }) => theme.colors[borderColor]};
+  border: ${({ borderColor }) =>
+    borderColor && `1px solid ${theme.colors[borderColor]}`};
```

Comment:
if we remove the border for the gradient one I think we still need to take into consideration the size of the element? If we wanna keep always the same height I think we either need to tweak the height or have some sort of transparent border?